sc/src/main/webapp/WEB-INF/tiles/views.xml

```diff
@@ -11,7 +11,7 @@
                 <put-attribute name="home-hero" value="/WEB-INF/jsp/tiles/views/home/hero.jsp"/>
                 <put-attribute name="search" value="/WEB-INF/jsp/tiles/views/home/search.jsp"/>
                 <put-attribute name="experiment-list-latest-box"
```

Comment:
This section of the JSP should be renamed, since it’s not only the the latest experiments.